SECURITY.md

```diff
@@ -0,0 +1,13 @@
+## Report a security issue
+
+The Tokio project team welcomes security reports and is committed to providing prompt attention to security issues. Security issues should be reported privately via [security@tokio.rs](mailto:security@tokio.rs). Security issues should not be reported via the public Github Issue tracker.
+
+## Vulnerability coordination
+
+Remediation of security vulnerabilities is prioritized by the project team. The project team coordinates remediation with third-party project stakeholders via [Github Security Advisories](https://help.github.com/en/github/managing-security-vulnerabilities/about-github-security-advisories). Third-party stakeholders may include the reporter of the issue, affected direct or indirect users of Tokio, and maintainers of upstream dependencies if applicable.
+
+Downstream project maintainers and Tokio users can request participation in coordination of applicable security issues by sending your contact email address, Github username(s) and any other salient information to [security@tokio.rs](mailto:security@tokio.rs). Participation in security issue coordination processes is at the discretion of the Tokio team.
```

Comment:
So, if a third party company wishes to be notified early of security issues, they would send an email to `security@tokio.rs`?

I am unclear on whether this is on a *per issue* basis or for all issues.

Reply:
We can merge and clarify in a follow up too.

Reply:
Good question Carl; this was left ambiguous here. I've seen both approaches work in practice. Many projects maintain a small list of trusted users that get all advanced notices; other orgs selectively disclose information on a case-by-case basis to reduce the likelihood of accidental advanced vuln disclosure (e.g. CERT/CC). Do you have a preference? We could always try something, see what works best, and iterate.